petab/lint.py

```diff
@@ -172,11 +172,12 @@ def check_parameterBounds(parameter_df):
     in the parameter table.
     """
     for element in range(len(parameter_df['lowerBound'])):
-        if not parameter_df['lowerBound'][element] \
-                <= parameter_df['upperBound'][element]:
-            raise AssertionError(
-                f"lowerbound larger than upperBound in parameterId "
-                f"{parameter_df.index[element]}.")
+        if int(parameter_df['estimate'][element]) is True:
```

Comment:
`int(*) is True` is never true. I think `1` is currently used.

Check `'estimate' in parameter_df` first

Reply:
in `check_parameter_df` it is already called first `assert_parameter_estimate_is_boolean(df)` and then `check_parameter_bounds(df)`

Reply:
If you do int('1') is resulting in True
`>>> int('1') == True
True`

Reply:
Yes, but `x == True` does not imply that `x is True == True`. The latter also checks for matching types.

Reply:
I replaced "is" with "=="